lms/static/js/fixtures/discovery.html

```diff
@@ -0,0 +1,52 @@
+<section class="courses-container">
```

Comment:
Since all your templates are underscore templates, would it be possible to load those directly in the test suite?  Here's an example that uses template helper utility methods to do this: https://github.com/edx/edx-platform/blob/master/lms/static/js/spec/student_account/login_spec.js#L137-L139